### PR TITLE
Adapt wording around JSON Logs Preprocessing in Pipelines

### PR DESCRIPTION
**Pull request by pcarioufr**
### Preview

https://docs-staging.datadoghq.com/pcarioufr/json_logs_preprocessing_rewording/logs/processing/#json-logs-preprocessing

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/processing/_index.md [hunk 1]
```diff
@@ -72,9 +72,36 @@ If you want to learn more about pure parsing possibilities of the Datadog applic
 For optimal usage of the Log Management solution, Datadog recommends using at most 20 processors per pipeline and 10 parsing rules within a grok processor. 
 Datadog reserves the right to disable underperforming parsing rules, processors, or pipelines that might impact Datadog's service performance.
 
-## Reserved attributes
+## JSON Logs Preprocessing
+
+JSON Logs preprocessing comes with a default configuration that work for standard log forwarders. Edit this configuration at any time to adapt to custom or specifc log forwarding approaches. To change the default values, go to the [Configuration page][5] and edit the `JSON Logs Preprocessing`:
+
+{{< img src="logs/processing/json_logs_preprocessing.gif" alt="JSON Logs Preprocessing Tile"  style="width:80%;">}}
```
Comment (pcarioufr) [inline content/en/logs/processing/_index.md:79]: I don't get why this one is broken in preview (although all CI tests passed).

+ The alternative text is still Reserved Attribute Tile 🤔 , which is the alternative text for the image we had in the previous version.

File: content/en/logs/processing/_index.md [hunk 2]
```diff
@@ -72,9 +72,35 @@ If you want to learn more about pure parsing possibilities of the Datadog applic
 For optimal usage of the Log Management solution, Datadog recommends using at most 20 processors per pipeline and 10 parsing rules within a grok processor. 
 Datadog reserves the right to disable underperforming parsing rules, processors, or pipelines that might impact Datadog's service performance.
 
-## Reserved attributes
 
-If your logs are formatted as JSON, be aware that some attributes are reserved for use by Datadog and are faceted by default:
+## JSON Logs Preprocessing
+
+JSON Logs preprocessing applies on all logs before they actually enter [Log Pipelines][1] processing. Preprocessing runs a series of operation based on reserved attributes:
```
Comment (cswatt) [inline content/en/logs/processing/_index.md:78]:
```suggestion
JSON Logs preprocessing applies on all logs before they actually enter [Log Pipelines][1] processing. Preprocessing runs a series of operations based on reserved attributes:
```

File: content/en/logs/processing/_index.md [hunk 3]
```diff
@@ -72,9 +72,35 @@ If you want to learn more about pure parsing possibilities of the Datadog applic
...
+
+* Trigger new [log integrations][16] based on the **source** of incoming logs.
+* Append incoming logs with all [**host** tags][22].
+* Apply Reserved Attribute Remappers Processors (namely [**Date** Remapper][12], [**Status** Remapper][15], [**Service** Remapper][19], [**Message** Remapper][20] and [**Trace ID** Remapper][21]) for the related JSON attributes of all incoming JSON logs.
```
Comment (cswatt) [inline content/en/logs/processing/_index.md:82]:
```suggestion
* Apply reserved attribute remapper processors (namely [**date** remapper][12], [**status** remapper][15], [**service** remapper][19], [**message** remapper][20] and [**trace ID** remapper][21]) for the related JSON attributes of all incoming JSON logs.
```

File: content/en/logs/processing/_index.md [hunk 4]
```diff
@@ -72,9 +72,35 @@ If you want to learn more about pure parsing possibilities of the Datadog applic
...
+
+JSON Logs preprocessing comes with a default configuration that work for standard log forwarders. Edit this configuration at any time to adapt to custom or specifc log forwarding approaches. To change the default values, go to the [Configuration page][5] and edit the `JSON Logs Preprocessing`:
```
Comment (cswatt) [inline content/en/logs/processing/_index.md:84]:
```suggestion
JSON Logs pre processing comes with a default configuration that works for standard log forwarders. Edit this configuration at any time to adapt to custom or specific log forwarding approaches. To change the default values, go to the [configuration page][5] and edit `Pre processing for JSON logs`:
```

File: content/en/logs/processing/_index.md [hunk 5]
```diff
@@ -72,9 +72,35 @@ If you want to learn more about pure parsing possibilities of the Datadog applic
 For optimal usage of the Log Management solution, Datadog recommends using at most 20 processors per pipeline and 10 parsing rules within a grok processor. 
 Datadog reserves the right to disable underperforming parsing rules, processors, or pipelines that might impact Datadog's service performance.
 
-## Reserved attributes
 
-If your logs are formatted as JSON, be aware that some attributes are reserved for use by Datadog and are faceted by default:
+## JSON Logs Preprocessing
```
Comment (cswatt) [inline content/en/logs/processing/_index.md:76]:
```suggestion
## JSON Logs Pre processing
```

File: content/en/logs/processing/_index.md (hunk 2 above)
Comment (cswatt) [inline content/en/logs/processing/_index.md:78]:
```suggestion
JSON Logs pre processing applies on all logs before they actually enter [Log Pipelines][1] processing. Pre processing runs a series of operation based on reserved attributes:
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**